server/mldb_server.h

```diff
@@ -142,6 +143,28 @@ struct MldbServer: public ServicePeer, public EventRecorder {
     Utf8String prefixUrl(Utf8String url) const;
     std::string prefixUrl(std::string url) const;
     std::string prefixUrl(const char* url) const;
+    
+    InProcessRestConnection mldbPerform(
```

Comment:
I agree that "mldbGet" is "ugly". With what would you go? I don't like "get" either because since this class is a server it's unclear what it would be. A couple of alternatives:
* getFromRoute, postToRoute, ...
* getFromUri, postToUri, ...
* getResource, postResource, ...

We can involve the nameing dept. if needed. ;)